arcade/backgrounds.py

```diff
@@ -0,0 +1,408 @@
+from PIL import Image
+
+from arcade.context import ArcadeContext
+from arcade.window_commands import get_window
+from arcade.resources import resolve_resource_path
+import arcade.gl as gl
+from pyglet.math import Mat3
+
+
+class BackgroundTexture:
```

Comment:
Possibly break this module into smaller parts so we can build on it easier. `arcade/backgrounsd` directly with an `__init__py` importing the classes located in sub-modules.